### ISSUE TITLE
Allow the override of the parameter "--version" in the CLI

### ISSUE THREAD

**Issue by pleminh** [absolunet/node-ioc#2]
Currently, we cannot use the variables 'version' in the CLI settings.  

We should allow the use of the version variable in the parameters.

Thank you

**Comment by thierry2015** [comment]
I agree it should be overridable by any command (it's been 1 year, dunno if you are still maintaining it though 😉)
